es-models/case_centric/case_centric.mapping.yaml

```diff
@@ -2,9 +2,6 @@ _all:
   enabled: false
 _size:
   enabled: true
-_source:
```

Comment:
Sorry, just noticed this. I don't think we want to remove these source exclusions.

Reply:
thanks, I resolved it

Reply:
Could you please revert the changes to the `_source`? We will want to continue to exclude the nested gene data from case centric and the nested case data from gene centric.

Reply:
I reverted it